src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/TestingControllers/ProcessControllerTest.cs

```diff
@@ -195,9 +196,9 @@ private HttpClient GetTestClient(IInstanceEventRepository instanceEventRepositor
                 // No setup required for these services. They are not in use by the ApplicationController
                 Mock<IApplicationRepository> applicationRepository = new Mock<IApplicationRepository>();
                 Mock<IDataRepository> dataRepository = new Mock<IDataRepository>();
-                Mock<IInstanceRepository> instanceRepository = new Mock<IInstanceRepository>();
                 Mock<ISasTokenProvider> sasTokenProvider = new Mock<ISasTokenProvider>();
                 Mock<IKeyVaultClientWrapper> keyVaultWrapper = new Mock<IKeyVaultClientWrapper>();
+                Mock<IPartiesWithInstancesClient> partiesWrapper = new Mock<IPartiesWithInstancesClient>();
```

Comment:
I guess this naming might be problematic if we were to require more parties resources in the future, but seems okay for now.